src/test/groovy/org/prebid/server/functional/tests/StoredResponseSpec.groovy

```diff
@@ -6,13 +6,13 @@ import org.prebid.server.functional.model.request.auction.StoredAuctionResponse
 import org.prebid.server.functional.model.response.auction.BidResponse
 import org.prebid.server.functional.testcontainers.PBSTest
 import org.prebid.server.functional.util.PBSUtils
-import spock.lang.PendingFeature
+
+import static org.prebid.server.functional.model.response.auction.ErrorType.PREBID
 
 @PBSTest
 class StoredResponseSpec extends BaseSpec {
 
-    @PendingFeature
-    def "PBS should not fail auction with storedAuctionResponse when request bidder params doesn't satisfy json-schema"() {
+    def "PBS should fail auction with storedAuctionResponse when request bidder params doesn't satisfy json-schema"() {
```

Comment:
i'd say it should not fail the auction. Let's add another bidder (for same or another `Imp`) with valid params and check the auction completed successfully. 
PS. For "another bidder" i mean in-request alias for `generic` bidder.